src/plasma/plasma_store.cc

```diff
@@ -44,8 +47,24 @@ void *dlmemalign(size_t alignment, size_t bytes);
 void dlfree(void *);
 }
 
+namespace std {
+template <>
+struct hash<UniqueID> {
+  size_t operator()(const UniqueID &unique_id) const {
+    return *reinterpret_cast<const size_t *>(unique_id.id + UNIQUE_ID_SIZE -
+                                             sizeof(size_t));
```

Comment:
I think the intention was that std::hash\<UniqueId\>() shouldn't return the same value for consecutive IDs, which I believe Philipp said was occurring somewhere. But actually, if unique_id.id is treated as a little-endian integer and then incremented, getting the last bytes in this way would cause the hash of consecutive IDs to usually be the same.